jaeger-core/src/test/java/io/jaegertracing/internal/senders/SenderResolverTest.java

```diff
@@ -53,6 +54,18 @@ public void testFaultySenderFactory() throws Exception {
         sender instanceof NoopSender);
   }
 
+  @Test
+  public void testNoopSenderFactory() throws Exception {
```

Comment:
Think it may be better to test this with a second variant of the `testMultipleImplementationsNotAmbiguous` - this test currently retrieves the first sender - but have one that specifies the second sender - but not using the `NoopSender` as an option, as this is also used to indicate no valid sender.

Reply:
Added a new assertion on `testMultipleImplementationsNotAmbiguous` to cover this case.